axelrod/fingerprint.py

```diff
@@ -53,110 +54,175 @@ def create_points(step, progress_bar=True):
 
-    @staticmethod
-    def create_jossann(point, probe):
-        """Creates a JossAnn probe player that matches the Point.
+    If the coordinates of point sums to more than 1 the parameters are
+    flipped and subtracted from 1 to give meaningful probabilities. We also
+    use the Dual of the probe. This is outlined further in [Ashlock2010]_.
 
-        If the coordinates of point sums to more than 1 the parameters are
-        flipped and subtracted from 1 to give meaningful probabilities. We also
-        use the Dual of the probe. This is outlined further in [Ashlock2010]_.
+    Parameters
+    ----------
+    point : Point
+    probe : class
+        A class that must be descended from axelrod.strategies
 
-        Parameters
-        ----------
-        point : Point
-        probe : class
-            A class that must be descended from axelrod.strategies
+    Returns
+    ----------
+    joss_ann: Joss-AnnTitForTat object
+        `JossAnnTransformer` with parameters that correspond to `point`.
+    """
+    x, y = point
 
-        Returns
-        ----------
-        joss_ann: Joss-AnnTitForTat object
-            `JossAnnTransformer` with parameters that correspond to `point`.
-        """
-        x, y = point
+    if isinstance(probe, axl.Player):
+        init_kwargs = probe.init_kwargs
+        probe = probe.__class__
+    else:
+        init_kwargs = {}
 
-        if isinstance(probe, axl.Player):
-            init_kwargs = probe.init_kwargs
-            probe = probe.__class__
-        else:
-            init_kwargs = {}
+    if x + y >= 1:
+        joss_ann = DualTransformer()(
+            JossAnnTransformer((1 - x, 1 - y))(probe))(**init_kwargs)
+    else:
+        joss_ann = JossAnnTransformer((x, y))(probe)(**init_kwargs)
+    return joss_ann
 
-        if x + y >= 1:
-            joss_ann = DualTransformer()(
-                JossAnnTransformer((1 - x, 1 - y))(probe))(**init_kwargs)
-        else:
-            joss_ann = JossAnnTransformer((x, y))(probe)(**init_kwargs)
-        return joss_ann
 
-    @staticmethod
-    def create_edges(points, progress_bar=True):
-        """Creates a set of edges for a spatial tournament.
+def create_probes(probe: Union[type, Player], points: list,
+                  progress_bar: bool = True) -> List[Player]:
+    """Creates a set of probe strategies over the unit square.
 
-        Constructs edges that correspond to `points`. All edges begin at 0, and
-        connect to the index +1 of the probe.
+    Constructs probe strategies that correspond to points with coordinates
+    (x, y). The probes are created using the `JossAnnTransformer`.
 
-        Parameters
-        ----------
-        points : list
-            of Point objects with coordinates (x, y)
-        progress_bar : bool
-            Whether or not to create a progress bar which will be updated
+    Parameters
+    ----------
+    probe : class
+        A class that must be descended from axelrod.strategies.
```

Comment:
since this takes either a class or an instance, perhaps it should mirror the docstring for `AshlockFingerprint.__init__`, 
which is

```
probe : class or instance
            A class that must be descended from axelrod.Player or an instance of
            axelrod.Player.
```